src/psyclone/transformations.py

```diff
@@ -1692,32 +1614,6 @@ def name(self):
         '''
         return "OMPMasterTrans"
 
-    def apply(self, node_list, options=None):
-        '''Apply the OMPMasterTrans transformation to the specified node in a
-        Schedule.
-
-        At code-generation time this node must be within (i.e. a child of)
-        an OpenMP PARALLEL region. Code generation happens when
-        :py:meth:`OMPMasterDirective.gen_code` is called, or when the PSyIR
-        tree is given to a backend.
-
-        :param node_list: the supplied node or node list to which we will \
-                          apply the OMPMasterTrans transformation
-        :type node_list: (a list of) :py:class:`psyclone.psyir.nodes.Node`
-        :param options: a list with options for transformations \
-                        and validation.
-        :type options: a dict of string:values or None
-
-        :returns: 2-tuple of new schedule and memento of transform.
-        :rtype: (:py:class:`psyclone.psyir.nodes.Schedule`,
-                 :py:class:`psyclone.undoredo.Memento`)
-
-        '''
-        if not options:
-            options = {}
-
-        return super(OMPMasterTrans, self).apply(node_list, options)
-
```

Comment:
While technically this is of course correct, I think we are losing some documentation here - as far as I can see the fact that this transformation must be within an OpenMP parallel region is not anywhere else. Either add this to the class documentation or bring this 'wrapper' function back (and maybe add a comment that this function is only for documentation purposes - Andy suggested something similar for me in the past as well).

Reply:
I documented this is the class docstring, I think it makes sense because there is also the example of applying both transformations there.

Reply:
Thanks, perfect solution.